dozer-ingestion/src/ingestion/ingestor.rs

```diff
@@ -80,6 +80,7 @@ impl Ingestor {
             }
             IngestionMessage::Commit(_event) => {}
             IngestionMessage::Begin() => {}
+            IngestionMessage::SnapshottingDone => {}
```

Comment:
Just making a mark here so later I know where to start passing along this message